app/views/user_mailer/member_invitation.html.haml

```diff
@@ -0,0 +1,4 @@
+Hi, my name is #{@member.name},
+%p You where invited to subscribe to website '#{@website[:domain]}'.
+%p
+  To accept invitation, just follow this link: #{link_to 'Accept Invitation', invitation_url(@website[:id]) + '?token=' + @token + '&email=' + @email} and complete the registration form.
```

Comment:
`invitation_url(id: @website[:id], token: @token, email: @email)` ..... sau `website_id in loc de id` nu stiu ce ruta ai de fapt.